sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -76,10 +79,33 @@ export interface DecryptResult {
 }
 
 // @public
-export interface DeletedKey extends Key {
-    readonly deletedDate?: Date;
-    readonly recoveryId?: string;
-    readonly scheduledPurgeDate?: Date;
+export interface DeletedKey {
+    keyMaterial?: JsonWebKey;
+    properties: KeyProperties & {
+        readonly recoveryId?: string;
+        readonly scheduledPurgeDate?: Date;
+        readonly deletedDate?: Date;
```

Comment:
Should be `deletedOn`.

Reply:
This won't be done as part of this PR. The issue tracking this change is #5577